qa/c8-orchestration-cluster-e2e-test-suite/tests/api/v2/incident/incident-search-api.spec.ts

```diff
@@ -0,0 +1,343 @@
+                    expect(item.errorType).toBe('EXTRACT_VALUE_ERROR');
+                });
+            }).toPass(defaultAssertionOptions);
+        });
+
+        await test.step('Search for CALLED_DECISION_ERROR incidents', async () => {
+            await expect(async () => {
+                const res = await request.post(buildUrl(INCIDENT_SEARCH_ENDPOINT), {
+                    headers: jsonHeaders(),
+                    data: {
+                    filter: {
+                        errorType: 'CALLED_DECISION_ERROR',
+                        processInstanceKey: localState['processInstanceKey'] as string,
+                    },
+                    },
+                });
+
+                await assertStatusCode(res, 200);
+                await validateResponse(
+                    {
+                    path: INCIDENT_SEARCH_ENDPOINT,
+                    method: 'POST',
+                    status: '200',
+                    },
+                    res,
+                );
+
+                const body = await res.json();
+                expect(body.page.totalItems).toEqual(1);
+                expect(body.items.length).toBeGreaterThan(0);
+                body.items.forEach((item: Record<string, unknown>) => {
+                    expect(item.errorType).toBe('CALLED_DECISION_ERROR');
+                });
+            }).toPass(defaultAssertionOptions);
+        });
+    });
+
+    test('Search Incidents Unauthorized', async ({request}) => {
+        await expect(async () => {
+            const res = await request.post(buildUrl(INCIDENT_SEARCH_ENDPOINT), {
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                data: {},
+            });
+
+            await assertUnauthorizedRequest(res);
+        }).toPass(defaultAssertionOptions);
+    });
+
+    test('Search Incidents Invalid Filter', async ({request}) => {
+        await expect(async () => {
+            const res = await request.post(buildUrl(INCIDENT_SEARCH_ENDPOINT), {
+                headers: jsonHeaders(),
+                data: {
+                filter: {
+                    meow: 'meowValue',
+                },
+                },
+            });
+
+            await assertBadRequest(
+                res,
+                'Request property [filter.meow] cannot be parsed',
+            );
+        }).toPass(defaultAssertionOptions);
+    });
+
+    test('Search Incidents Empty Result success', async ({request}) => {
+        const localState: Record<string, unknown> = {};
+        await setupIncidentTest(localState, request);
+        processInstanceKeys.push(localState['processInstanceKey'] as string);
+
+        await expect(async () => {
+            const res = await request.post(buildUrl(INCIDENT_SEARCH_ENDPOINT), {
+                headers: jsonHeaders(),
+                data: {
+                filter: {
+                    errorType: 'JOB_NO_RETRIES',
```

Comment:
maybe there is a better way to filter for something you are sure it's not there. Like a overspecific name or so on. Another option is to add the processInstanceKey to the filter. Right now if another tests produces a process with an errorType 'JOB_NO_RETRIES' this tests fails.